index.js

```diff
@@ -3,6 +3,124 @@ const nav = document.querySelector('.nav-wrapper');
 const closeBtn = document.querySelector('.close-menu');
 const ancreLinks = nav.getElementsByTagName('a');
 
+const theModal = document.querySelector('#myModal');
+const closeModal = document.querySelector('.closeModal');
+const worksSection = document.querySelector('.work-container');
+
+const data = [
+  {
+    name: 'Tonic',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi architecto 
+    mollitia, earum fugiat possimus expedita! Qui nostrum blanditiis est, cupiditate autem 
+    fugiat non consequatur consectetur itaque dolor, quo deserunt sequi.
+    Consequuntur placeat, delectus eius nemo veritatis eaque quas. Nulla minima, harum nobis 
+    odio, in non placeat voluptas provident vitae, quaerat maxime minus amet. Libero nisi 
+    maiores molestias repudiandae natus provident.`,
+    featuredImage: './image/snapshoot.png',
+    technologies: ['html', 'css', 'javaScript'],
+    linkToLive: '',
+    linkToSource: '',
+    cardNo: '#first-card',
+  },
+  {
+    name: 'Multi-Post Stories',
+    shortDescription: 'A daily selection of privately personalized reads; no accounts or sign-ups required.',
+    description: `Lorem ipsum dolor sit amet consectetur adipisicing elit. Quasi architecto 
+    mollitia, earum fugiat possimus expedita! Qui nostrum blanditiis est, cupiditate autem 
+    fugiat non consequatur consectetur itaque dolor, quo deserunt sequi.
+    Consequuntur placeat, delectus eius nemo veritatis eaque quas. Nulla minima, harum nobis 
+    odio, in non placeat voluptas provident vitae, quaerat maxime minus amet. Libero nisi 
+    maiores molestias repudiandae natus provident.`,
+    featuredImage: './image/snapshoot-1.png',
+    technologies: ['html', 'css', 'javaScript'],
+    linkToLive: '',
+    linkToSource: '',
```

Comment:
[OPTIONAL] I think it will be better to make the see source and see live buttons on our modal functional using the live version link and the repository link of your portfolio for see live and see source button respectively.